software/contrib/egressus_melodiam.md

```diff
@@ -0,0 +1,87 @@
+# Egressus Melodiam (Stepped Melody)
+
+author: Nik Ansell (github.com/gamecat69)
+date: 02-Feb-24
+labels: clocked lfo, sequencer, CV, randomness
+
+Clockable and free-running LFO and random CV pattern generator.
+Six different wave shapes either define the LFO shape or the slew shape between CV pattern steps.
+Use it to generate variable length melodies (using a quantizer), LFOs or wierd and wonderful control voltages.   
+
+Demo video: TBC
+
+# Inputs, Outputs and Controls
+
+digital_in: clock in
+analog_in: not used

```

Comment:
I have this option in the potential future work section in the python file header :).
It would be easy to implement, but I would need to test it fully to make sure it does not add any more processing latency.